<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="image_6">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>260</y>
     <width>175</width>
     <height>175</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/100071.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="image_1">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>20</y>
     <width>175</width>
     <height>175</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/104.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="image_3">
   <property name="geometry">
    <rect>
     <x>472</x>
     <y>20</y>
     <width>175</width>
     <height>175</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/1119.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="image_2">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>20</y>
     <width>175</width>
     <height>175</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/111.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="image_5">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>260</y>
     <width>175</width>
     <height>175</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/12113.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="image_7">
   <property name="geometry">
    <rect>
     <x>472</x>
     <y>260</y>
     <width>175</width>
     <height>175</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/100132.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="image_4">
   <property name="geometry">
    <rect>
     <x>692</x>
     <y>20</y>
     <width>175</width>
     <height>175</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/10285.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="image_8">
   <property name="geometry">
    <rect>
     <x>692</x>
     <y>260</y>
     <width>175</width>
     <height>175</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/653.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>files/Bugs_blurred.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name_1">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>195</y>
     <width>175</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 64);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name_5">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>435</y>
     <width>175</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 64);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name_2">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>195</y>
     <width>175</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 64);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name_6">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>435</y>
     <width>175</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 64);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name_3">
   <property name="geometry">
    <rect>
     <x>472</x>
     <y>195</y>
     <width>175</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 64);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name_7">
   <property name="geometry">
    <rect>
     <x>472</x>
     <y>435</y>
     <width>175</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 64);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name_4">
   <property name="geometry">
    <rect>
     <x>692</x>
     <y>195</y>
     <width>175</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 64);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name_8">
   <property name="geometry">
    <rect>
     <x>692</x>
     <y>435</y>
     <width>175</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 64);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="linkButton_1">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>19</y>
     <width>177</width>
     <height>242</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="linkButton_2">
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>19</y>
     <width>177</width>
     <height>242</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="linkButton_3">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>19</y>
     <width>177</width>
     <height>242</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="linkButton_4">
   <property name="geometry">
    <rect>
     <x>691</x>
     <y>19</y>
     <width>177</width>
     <height>242</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="linkButton_5">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>259</y>
     <width>177</width>
     <height>242</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="linkButton_6">
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>259</y>
     <width>177</width>
     <height>242</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="linkButton_7">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>259</y>
     <width>177</width>
     <height>242</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="linkButton_8">
   <property name="geometry">
    <rect>
     <x>691</x>
     <y>259</y>
     <width>177</width>
     <height>242</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>image_6</zorder>
  <zorder>image_1</zorder>
  <zorder>image_3</zorder>
  <zorder>image_2</zorder>
  <zorder>image_5</zorder>
  <zorder>image_7</zorder>
  <zorder>image_4</zorder>
  <zorder>image_8</zorder>
  <zorder>name_1</zorder>
  <zorder>name_5</zorder>
  <zorder>name_2</zorder>
  <zorder>name_6</zorder>
  <zorder>name_3</zorder>
  <zorder>name_7</zorder>
  <zorder>name_4</zorder>
  <zorder>name_8</zorder>
  <zorder>linkButton_1</zorder>
  <zorder>linkButton_2</zorder>
  <zorder>linkButton_3</zorder>
  <zorder>linkButton_4</zorder>
  <zorder>linkButton_5</zorder>
  <zorder>linkButton_6</zorder>
  <zorder>linkButton_7</zorder>
  <zorder>linkButton_8</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
